docs/sources/awssqs.md

```diff
@@ -1,113 +1,93 @@
 # AWS SQS event source for Knative Eventing
```

Comment:
The title can also be changed to match [existing doc pages](https://docs.triggermesh.io/sources/awscodecommit/) (no mention of Knative needed here):
```suggestion
# Event source for AWS SQS
```